client/app/admin/admin.html

```diff
@@ -45,6 +45,7 @@
             <td colspan="2">Small Group</td>
             <td colspan="2">Contributions</td>
             <td>Toggle</td>
```

Comment:
You should change `<td>Toggle</td>` to `<td colspan="2">Edit</td>`, then move  `<td>Attendance</td>` to the second header line after  `<td sort-arrow sortby='active' text='Active' sortorder='sortorder'></td>`.